roles/openshift_facts/library/openshift_facts.py

```diff
@@ -558,17 +597,17 @@ def set_deployment_facts_if_unset(facts):
             service_type = 'atomic-openshift'
             if deployment_type == 'origin':
                 service_type = 'origin'
-            elif deployment_type in ['enterprise', 'online']:
+            elif deployment_type in ['openshift-enterprise', 'enterprise', 'online']:
```

Comment:
Can you clarify, these three lines should revert, and we leave it with "origin" dirs if openshift-enterprise or atomic-enterprise? Do I have that right?